src/main.py

```diff
@@ -30,6 +31,17 @@
 cache_service = CacheService(settings)
 
 
+@app.exception_handler(FlagsmithUnknownKeyError)
+async def unknown_key_error(request, exc):
+    return JSONResponse(
+        status_code=401,
+        content={
+            "status": "forbidden",
```

Comment:
Sorry, I missed the status in my last review comment. We should update this to `unauthorized` as well